README.md

```diff
@@ -8,6 +8,8 @@ To allow users to provide feedback about the ONS website
 
 ## Dependencies
 
+- dp-design-system - Serves CSS and JS for the page
+- Run a local mail server to receive feedback 
```

Comment:
given we can switch between the two, we either need local mailserver *or* feedback-api right? Might want a link to the dp-compose stack too.